spec/spec_helper.rb

```diff
@@ -1,14 +1,11 @@
 # Make sure specs run with the definitions from test.rb
 ENV['ROBOT_ENVIRONMENT'] = 'test'
 
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
+
 require 'coveralls'
 Coveralls.wear!
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-  CodeClimate::TestReporter::Formatter,
-  Coveralls::SimpleCov::Formatter
-]
```

Comment:
See https://docs.codeclimate.com/docs/test-coverage-troubleshooting-tips#section--send-coverage-to-both-code-climate-and-coveralls-ruby-only-

...